src/components/ImportForm/ComponentSection/SourceSection.tsx

```diff
@@ -39,7 +39,7 @@ export const SourceSection = () => {
         let name: string;
         try {
           parsed = GitUrlParse(event.target?.value ?? '');
-          setFieldValue('gitURLAnnotation', parsed?.resource);
+          setFieldValue('gitURLAnnotation', `https://${parsed?.resource}`);
```

Comment:
Maybe avoid hardcoding
```suggestion
          setFieldValue('gitURLAnnotation', `${parsed?.protocol}://${parsed?.resource}`);
```